app/controllers/api/v1/sessions_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Api
+  module V1
+    class SessionsController < DeviseTokenAuth::SessionsController
+      include ActAsApi
+    end
```

Comment:
I think you can do something similar as you did in the `RegistrationController`, you can override the method `create` and also `sign_in_params`, check at this:
[SessionController](https://github.com/plataformatec/devise/blob/master/app/controllers/devise/sessions_controller.rb)

Then you can remove `devise_parameter_sanitizer.permit(:sign_in, keys: %i[email])` above in the `ApiController`

Reply:
I dont fully understand the create method on the controller, time to come to work 😏